packages/neo-one-client/src/Client.ts

```diff
@@ -145,6 +149,7 @@ export class Client<TUserAccountProviders extends { readonly [K in string]: User
     return [...new Set(providers.reduce((acc: NetworkType[], provider) => acc.concat(provider.getNetworks()), []))];
   }
 
+  // tslint:disable-next-line readonly-array no-any
```

Comment:
Remove

Reply:
The merge conflict was because of a source map argument getting added to a call that hadn't been rebased because I hadn't worked on it. I believe everything should be fixed now.